connector_ecommerce/models/product.py

```diff
@@ -29,19 +12,23 @@ class ProductTemplate(models.Model):
 
     # TODO implement set function and also support multi tax
     @api.one
-    @api.depends('taxes_id', 'taxes_id.group_id')
-    def _get_tax_group_id(self):
+    @api.depends('taxes_id', 'taxes_id.tax_group_id')
```

Comment:
That's not totally true. It would be true if the field `tax_group_id` is required, because always that you create a new record in taxes_id, the field "changes". But if you can create a record that doesn't have any value in `tax_group_id`, then the trigger won't be fired.